examples/complete/azure/advanced/aks-with-peering/hopsworks.tf

```diff
@@ -0,0 +1,305 @@
+
+
+// We should all external ACR and AKS references here since user's won't run the AKS part
```

Comment:
It's missing a verb after "should".
And I'm not sure what `user's won't run the AKS part` means

Reply:
fixed